gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/common/impl/ConfigStoreBackedTopology.java

```diff
@@ -0,0 +1,66 @@
+package gobblin.config.common.impl;
+
+import gobblin.config.store.api.ConfigKeyPath;
+import gobblin.config.store.api.ConfigStore;
+import gobblin.config.store.api.ConfigStoreWithImportedBy;
+import gobblin.config.store.api.ConfigStoreWithImportedByRecursively;
+import gobblin.config.store.api.ConfigStoreWithResolution;
+
+import java.util.Collection;
+import java.util.List;
+
+public class ConfigStoreBackedTopology implements ConfigStoreTopologyIntf{
+  
+  private final ConfigStore cs;
+  private final String version;
+  
+  public ConfigStoreBackedTopology(ConfigStore cs, String version){
+    this.cs = cs;
+    this.version = version;
+  }
+  
+  public ConfigStore getConfigStore(){
+    return this.cs;
+  }
+  
+  public String getVersion(){
+    return this.version;
+  }
+
+  @Override
+  public Collection<ConfigKeyPath> getChildren(ConfigKeyPath configKey) {
```

Comment:
nit: Please add the following javadoc to overridden methods:

/** {@inheritDoc} */